scss/_navbar.scss

```diff
@@ -1,29 +1,37 @@
-// Contents
-//
-// Navbar
-// Navbar brand
-// Navbar nav
-// Navbar text
-// Responsive navbar
-// Navbar position
-// Navbar themes
-
-
 // Navbar
 //
 // Provide a static navbar from which we expand to create full-width, fixed, and
 // other navbar variations.
 
 .navbar {
+  // scss-docs-start navbar-css-vars
+  --#{$variable-prefix}navbar-padding-x: #{if($navbar-padding-x == null, 0, $navbar-padding-x)};
```

Comment:
We end up with this `if` and the `escape-svg` function bellow, not sure if we want this to show like that?

Reply:
Yup, I want this here for now. I expect to make more use of the inline if statements and other functions like this to generate our CSS variables.

Reply:
Sorry I meant they show up in the docs variables too. I'm not sure we want that to happen :)